Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -830,11 +844,12 @@ void PluginView::pendingResourceRequestTimerFired()
         return;
 
     // This protector is needed to make sure the PluginView is not destroyed while it is still needed.
-    Ref protectThis { *this };
+    Ref protectedThis { *this };
 
     ASSERT(!m_stream);
-    m_stream = PluginView::Stream::create(this, *request);
-    m_stream->start();
+    Ref stream = PluginView::Stream::create(this, *request);
+    m_stream = stream.copyRef();
```

Comment:
Could we not do `m_stream = PluginView::Stream::create(this, *request);`? I assumed that would `template<typename X> RefPtr& operator=(Ref<X>&&);`

Reply:
The issue wasn't that assignment. The issue is that I need a `Ref` *on the stack* so I can call `stream->start();` safely on the next line.

Reply:
oh! Okay, cool